### PR TITLE
add keepalive_timeout flag

### PR DESCRIPTION
**Pull request by fenwuyaoji**
# Issue

[What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.]
Relevant to:
https://github.com/Project-OSRM/osrm-backend/issues/6673 Rquire a configurable parameter to set keepalive_timeout.

Please read our [documentation](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/releasing.md) on release and version management.
If your PR is still work in progress please attach the relevant label.

## Tasklist

 - [x] CHANGELOG.md entry ([How to write a changelog entry](http://keepachangelog.com/en/1.0.0/#how))
 - [x] update relevant [Wiki pages](https://github.com/Project-OSRM/osrm-backend/wiki)
 - [x] add tests (see [testing documentation](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/testing.md))
 - [x] review
 - [x] adjust for comments
 - [x] cherry pick to release branch

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Review by fenwuyaoji — COMMENTED** [review]
link the PR

**Comment by mjjbell** [comment]
Please run `scripts/format.sh` on your branch. Otherwise, looks good.

**Review by mjjbell — APPROVED** [review]

**Comment by fenwuyaoji** [comment]
@mjjbell Thanks for your comment, I have fixed the format problem, could you rerun the workflow?

**Comment by fenwuyaoji** [comment]
@mjjbell Sorry for the late response. I checked the page https://github.com/Project-OSRM/osrm-backend/blob/master/CONTRIBUTING.md and added 
.git/hooks/pre-push to my local env and all things look good in my env now. I hope this time can pass the workflow. Could you trigger the workflow again?

**Comment by fenwuyaoji** [comment]
@mjjbell Sorry to disturb you again. I saw there was one test case that didn't pass. I'm not familiar with the js test so I just copied and modified another one as mine. Now I notice that, if I want to throw an error, I should first define it in node_osrm_support.hpp. I checked the case and thought it unnecessary so I deleted it. Could you rerun it?

**Comment by fenwuyaoji** [comment]
@mjjbell Really appreciate! Have a good day~

File: test/nodejs/index.js [hunk 1]
```diff
@@ -131,6 +131,12 @@ test('constructor: throws if default_radius is not a number', function(assert) {
     assert.ok(new OSRM({algorithm: 'MLD', path: monaco_mld_path, default_radius: 'unlimited'}), 'Does accept unlimited');
 });
 
+test('constructor: takes a keepalive_timeout argument', function(assert) {
+    assert.plan(1);
+    var osrm = new OSRM({algorithm: 'MLD', path: monaco_mld_path, keepalive_timeout: 10});
```
Comment (mjjbell) [inline test/nodejs/index.js:136]: The Node bindings do not use the HTTP server, so the `keepalive_timeout` won't do anything here.
I would just remove the test.

Comment (fenwuyaoji) [reply test/nodejs/index.js]: Sure, I just remove it.

**Pull request merged**